tests/unit_tests/expected_md/summary_regressions_baselineisnotparent.md

```diff
@@ -0,0 +1,13 @@
+Contender commit `abc` had 2 regressions compared to its baseline commit.
+
+### Benchmarks with regressions:
+
+- Run ID [some_contender1](https://conbench/some_baseline1...some_contender1)
+  - `snappy, nyctaxi_sample, parquet, arrow`
```

Comment:
Wait where is it getting these? In the code it looked like this string is just called `benchmark` and coming from upstream somewhere, but AFAIK we don't necessarily store this in this form anywhere; the C++ benchmarks store parameters differently than Py/R. Read: Are there any assumptions here I may break?

Reply:
So this `"benchmark"` key is coming from the Conbench compare runs endpoint. Digging into the Conbench code, I think that's the `display_name`, which is set [here](https://github.com/conbench/conbench/blob/main/conbench/hacks.py#L22) on read, not stored. It looks like it defaults to the `name` tag if it exists, else it's concatenated from the tags that do exist.

I think this will always produce a nice little name for this markdown report, no matter what changes you make. And any changes you do make won't affect these unit tests because this response is mocked anyway.

Reply:
Cool! We should maybe add the keys in that, though that's beyond the scope of this PR

Reply:
> It looks like it defaults to the name tag if it exists, else it's concatenated from the tags that do exist.

Yeah this is the (probably underdocumented) behavior. This is intended to be a nice(ish) thing that will encapsulate the unique benchmark (case) and be something a person could read and figure out where to go from there.

> We should maybe add the keys in that, though that's beyond the scope of this PR

Agreed on both fronts — we might want to look through the issues or the git history on this front. I can't remember exactly if we had a long time ago talked about adding the keys (and then never got around to it) or if they were there originally and then we pulled them out for some reason — but I *do* remember talking about this in the early days of conbench

Reply:
Looks like the code does include the keys -- but only if the value is a number. 🙂

Reply:
I just dug through the git history and this keyless code has been around since the [original commit](https://github.com/conbench/conbench/commit/5ee1d6117cd774121280d0ae4468751be41acaa0#diff-82b207686450cdedf4f2a29b5431cf17d3aaccbfb1038fe24637eaed1671451cR19). I also couldn't find any issues about it. So maybe that discussion happened before we open-sourced the project.

Reply:
Made https://github.com/conbench/conbench/issues/400